src/config.rs

```diff
@@ -487,6 +490,7 @@ pub struct RaftDbConfig {
     pub use_direct_io_for_flush_and_compaction: bool,
     pub enable_pipelined_write: bool,
     pub allow_concurrent_memtable_write: bool,
+    pub manual_wal_flush: bool,
```

Comment:
I think we don't need this for raft because we need to flush for raft every time.

Reply:
raft log gc may not need flush

Reply:
do we use flush wal in raft log gc now?

Reply:
no, it just call db.write

Reply:
What's the purpose of this option in raftdb?